rpcd/playbooks/ceph-mon.yml

```diff
@@ -19,3 +19,18 @@
   max_fail_percentage: 0
   roles:
     - ceph-mon
+  tasks:
+    - name: Check if rbd pool exists and is empty
+      shell: rados -p rbd df | egrep '^rbd( +0){9}$'
+      run_once: true
+      register: rbd_pool_exists
```

Comment:
you either need an ignore_failure here or you'll need to use a ceph df | grep rbd.
Since the command fails when the rbd pool doesn't exist this will fail out.